### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by QZheng82** [Anura3D/Anura3D_OpenSource#50] [Q&A] [answered]
I would like to ask what does  "Particle   5325 from element  12132 lies outside the mesh"mean?

> My grid
![image](https://user-images.githubusercontent.com/119284513/204469838-de631acf-b36e-4d91-8949-b55425c4514b.png)

> ![image](https://user-images.githubusercontent.com/119284513/204469152-ef74d418-bb92-46d8-a647-abf553d942f2.png)

> My boundary condition
![image](https://user-images.githubusercontent.com/119284513/204470871-32f01d0f-9afc-446d-8ed3-d72ac84e2662.png)

**Comment by murphy-john** [comment, marked as answer]
This means the solution went unstable and one of the MPs had an updated position which was outside the mesh.
